src/main/java/org/akhq/repositories/RecordRepository.java

```diff
@@ -1268,5 +1273,14 @@ public TailOptions(String clusterId, List<String> topics) {
         private final long end;
         private final KafkaConsumer<byte[], byte[]> consumer;
     }
+
+    private void filterMessageLength(Record record) {
```

Comment:
I would prefer add a property `truncated` as bool on record to avoid to have translation on backend

Reply:
Do you refer to sending the entire message and only catching this property in front end?

Reply:
you truncate the message on the backend plus add a property to tell the message is truncated. 
The front will only add the ... too large message ... if truncated is true